src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -749,6 +740,11 @@ def load_arguments(self, _):
                           "To enable Multitenant Hosting Rights for Windows 10, use 'Windows_Client'. " \
                           "For more information see the Azure Windows VM online docs."
             c.argument('license_type', help=license_msg, arg_type=get_enum_type(['Windows_Server', 'Windows_Client', 'None']))
+            c.argument('priority', resource_type=ResourceType.MGMT_COMPUTE, min_api='2019-03-01',
```

Comment:
curious about the `min_api`. If the min_api is different in vm and vmss, should we keep them separated?

Reply:
Yes, we should. This the cause of error in Profile 2019-03-01.